src/SonarScanner.Shim/ProjectLoader.cs

```diff
@@ -30,12 +30,12 @@ public static class ProjectLoader
         public static IList<ProjectInfo> LoadFrom(string dumpFolderPath)
         {
             return Directory.GetDirectories(dumpFolderPath)
-                .Select(TryGetProjectInfo)
+                .Select(GetProjectInfo)
```

Comment:
My expectation of _GetXXX_ methods is that they will throw rather than return null.

I may well have written the original method. I don't find the _Try_ pattern particularly useful so I tended not to use it for private methods. What the method name was trying to convey is that "_this method might return null, so be prepared to handle nulls_".

I think _FindXXX_ is now a fairly common naming pattern for methods that might return null.

Reply:
`Find` is fine by me but doesn't look really C#. I tend to see more `GetXXXOrDefault`

Reply:
Also maybe that's my defensive coding style but if a method returns a class I always assume it can be null.